File: lib_network/src/main/java/com/caesarlib/network/bean/yesapi/ExtInfoData.java
package com.caesarlib.network.bean.yesapi;

import com.google.gson.Gson;

/**
 * 用户扩展信息
 * created by Caesar on 2019/5/29
 * email : 15757855271@163.com
 */
public class ExtInfoData {
    private String nickName;
    private int age;
    private int sex;
    private String province;
    private String city;
    private String area;
    private String headImg;

    public String getHeadImg() {
        return headImg;
    }

    public void setHeadImg(String headImg) {
        this.headImg = headImg;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public int getSex() {
        return sex;
    }

    public void setSex(int sex) {
        this.sex = sex;
    }

    public String getProvince() {
        return province;
    }

    public void setProvince(String province) {
        this.province = province;
    }

    public String getCity() {
        return city;
    }

    public void setCity(String city) {
        this.city = city;
    }

    public String getArea() {
        return area;
    }

    public void setArea(String area) {
        this.area = area;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }


    public String toJson() {
        return new Gson().toJson(this);
    }

}

File: lib_network/src/main/java/com/caesarlib/network/bean/yesapi/ExtInfoData.kt
package com.caesarlib.network.bean.yesapi

import com.google.gson.Gson

/**
 * 用户扩展信息
 * created by Caesar on 2019/5/29
 * email : 15757855271@163.com
 */
class ExtInfoData {
    //    var nickName: String? = null
//    var age: Int = 0
//    var sex: Int = 0
    var yesapi_avatar: String? = null
    var yesapi_nickname: String? = null
    var yesapi_sex: String? = null
    var yesapi_age: Int = 0
    var province: String? = null
    var city: String? = null
    var area: String? = null
//    var headImg: String? = null


    fun toJson(): String {
        return Gson().toJson(this)
    }

}
